doc/services/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -840,3 +842,197 @@ where:
     | "rc"             | :c:enum:`mcumgr_err_t` only appears if non-zero (error condition) when  |
     |                  | using SMP version 1 or for SMP errors when using SMP version 2.         |
     +------------------+-------------------------------------------------------------------------+
+
+Bootloader Information
+**********************
+
+Allows retrieving information about the on-board bootloader and its parameters.
+
+Bootloader Information Request
+==============================
+
+Bootloader information request header:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``0``  | ``0``        |  ``8``         |
+    +--------+--------------+----------------+
+
+CBOR data of request:
+
+.. code-block:: none
+
+    {
+        (str,opt)"query"  : (str)
+    }
+
+where:
+
+.. table::
+    :align: center
+
+    +--------------+-----------------------------------------------+
+    | "query"      | Is string representing query for parameters,  |
+    |              | with no restrictions how the query looks like |
+    |              | as processing of query is left for bootloader |
+    |              | backend.                                      |
+    |              | If there is no query, then response will      |
+    |              | return string identifying the bootloader.     |
+    +--------------+-----------------------------------------------+
+
+Bootloader Information Response
+===============================
+
+Bootloader information response header:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``1``  | ``0``        |  ``8``         |
+    +--------+--------------+----------------+
+
+In case when no "query" has been provided in request,
+CBOR data of response:
+
+.. code-block:: none
+
+    {
+        (str)"bootloader"      : (str)
+    }
+
+where:
+
+.. table::
+    :align: center
+
+    +--------------+-----------------------------------------------+
+    | "bootloader" | String representing bootloader name           |
+    +--------------+-----------------------------------------------+
+
+In case when "query" is provided:
+
+.. code-block:: none
+
+    {
+        (str,opt)<param_name>   : ()
+	...
+    }
+
+where:
+
+.. table::
+    :align: center
+
+    +------------------+-------------------------------------------------------------------------+
+    | <param_name>     | Response to "query". This is optional and may be left out in case when  |
+    |                  | query yields no response, but the "rc" code needs to `MGMT_ERR_ENOENT`. |
```

Comment:
Don't think this error code is right, it's `OS_MGMT_ERR_QUERY_YIELDS_NO_ANSWER`?